src/SFA.DAS.PSRService.Web/Controllers/ReportController.cs

```diff
@@ -154,6 +181,7 @@ public IActionResult Summary([FromRoute] string hashedEmployerAccountId, string
 
                 reportViewModel.IsValidForSubmission = reportViewModel.Report?.IsValidForSubmission() ?? false;
                 reportViewModel.Percentages = new PercentagesViewModel(reportViewModel.Report?.ReportingPercentages);
+                if (reportViewModel.Report.ReportingPercentagesSchools != null) reportViewModel.PercentagesSchools = new PercentagesViewModel(reportViewModel.Report?.ReportingPercentagesSchools);
```

Comment:
can we just check if the Report is null once rather than on every single line in this method?

Reply:
also inconsistently null checking `.Report` property